test/autorest/generated/morecustombaseurigroup/internal/morecustombaseurigroup/paths.go

```diff
@@ -6,18 +6,21 @@
 package morecustombaseurigroup
 
 import (
-	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 	"net/http"
 	"net/url"
 	"path"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 )
 
 type PathsOperations struct{}
 
 // GetEmptyCreateRequest creates the GetEmpty request.
-func (PathsOperations) GetEmptyCreateRequest(u url.URL, vault string, secret string, keyName string, keyVersion string) (*azcore.Request, error) {
-	u.Path = path.Join(u.Path, "/customuri/{subscriptionId}/{keyName}")
-	return azcore.NewRequest(http.MethodGet, u), nil
+func (PathsOperations) GetEmptyCreateRequest(u url.URL, vault string, secret string, keyName string, keyVersion string, subscriptionID string) (*azcore.Request, error) {
+	u.Path = path.Join(u.Path, "/customuri", subscriptionID, keyName)
```

Comment:
I like this solution.  Is there a bug tracking fixing this in the generator?

Reply:
I added it as part of my general list of bugs that I've found while doing the tests but I'll also add it's own specific one. 

Also for some reason both the DNS and subscriptionID param were not added. I wonder if it has to do with how they're referenced in the swagger.

Reply:
Here's the issue: #263

Reply:
For the missing parameters, my guess is they're defined as `client` which in track 1 meant they were parameters on the client constructor.  For the internal package we'll need to add them in the regular list of params.  And for the convenience layer we'll likely need to add them as constructor params.

Reply:
Ah probably. I'm going to add your last two sentences as comments in the issue.